execution/interface.go

```diff
@@ -57,10 +57,10 @@ type ExecutionSequencer interface {
 	SequenceDelayedMessage(message *arbostypes.L1IncomingMessage, delayedSeqNum uint64) error
 	NextDelayedMessageNumber() (uint64, error)
 	MarkFeedStart(to arbutil.MessageIndex)
+	SyncProgressMap() map[string]interface{}
```

Comment:
I really prefer the Synced/FullSyncProgressMap interface.
I realize that currently the implementation of these functions in execution sync-monitor is not great.. but I would still opt to use the better interface and fix the implementation later (or, if you want, you can fix it here :))

Reply:
I understand it now, by itself `SyncProgressMap` doesn't make much sense, whereas Synced and FullSyncProgressMap each have specific purpose and better to use them!